tap_zendesk/__init__.py

```diff
@@ -13,7 +14,15 @@
 REQUIRED_CONFIG_KEYS = [
     "start_date",
     "subdomain",
-    "access_token"
+]
+
+API_CONFIG_KEYS = REQUIRED_CONFIG_KEYS + [
```

Comment:
Adding a similar comment here about the mutual exclusivity and preference ordering of these two config sections would be nice.